src/demos/calcite-link.html

```diff
@@ -33,9 +33,23 @@ <h3>Set focus</h3>
     <calcite-button onclick="document.querySelector('#button-focus-example-2').setFocus()">focus a </calcite-button>
     <h4>Receive focus</h4>
 
-    <calcite-link id="button-focus-example-1">span to gain focus</calcite-link>
-    <br />
-    <calcite-link id="button-focus-example-2" href="http://google.com">anchor to gain focus</calcite-link>
+
+  <h3>Text sizes</h3>
+  <p style="font-size:10px"><calcite-link icon-start="information" icon-end="information">example text</calcite-link><br /></p>
```

Comment:
Can you add capitalized "Link" text and a filled icon to either start/end too, for additional demo/reference purposes?
```html
<calcite-link icon-start="information" icon-end="information-f">Link example text</calcite-link>
```